apps/L1E/instagram/instagram-frontend/src/app/(main)/(auth)/components/InputWithToggle.tsx

```
@@ -0,0 +1,27 @@
+import { Input } from "@/components/ui/input";
+import { Eye, EyeOff } from "lucide-react";
+import { useState } from "react";
+
+interface InputWithToggleProps extends React.InputHTMLAttributes<HTMLInputElement> {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-interface)